CHANGELOG.md

```diff
@@ -0,0 +1,28 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+### Changed
+
+## [0.1.1]
+
+### Added
+
+- Support installation of dbt versions >= 0.20.0 in a Python 3.9 environment.
+
+### Changed
+
+## [0.1.0] - 2021-04-16
```

Comment:
- I made the initial version 1.0.0 because I considered it fairly feature complete and usable at that point, and also to avoid the trap dbt has fallen into of not following [Semantic Versioning](https://semver.org/spec/v2.0.0.html) just because it's pre-v1.
- I just made a [v1.0.0 release](https://github.com/brooklyn-data/dbtenv/releases/tag/v1.0.0) in GitHub, and we should make such GitHub releases going forward (that'll be required for these comparison links to work).
- At first I thought you got the date wrong as to me it was 2021-04-15, but then I realized it was 2021-04-16 in your timezone.  I think we should standardize on a timezone to base release dates on, and to me the logical choice is UTC (so this would stay 2021-04-16).
```suggestion
## [1.0.0](https://github.com/brooklyn-data/dbtenv/releases/tag/v1.0.0) - 2021-04-16
```